src/main/java/com/stuypulse/robot/subsystems/ColorSensor.java

```diff
@@ -50,7 +53,16 @@
 
     public ColorSensor() {
         colorSensor = new ColorSensorV3(Ports.COLOR_SENSOR);
-        getUpdateFromDriverStation();
+        configureAllianceColor();
+        updateAllianceColor();
+    }
+    
+    public void configureAllianceColor() {
+        Settings.ColorSensor.ALLIANCE_COLOR.setDefaultOption("Use FMS", Alliance.Invalid);
+        Settings.ColorSensor.ALLIANCE_COLOR.addOption("Red", Alliance.Red);
```

Comment:
put this in a static method getAllianceChooser() in Settings and 
just say SendableChooser<...> AUTON_CHOOSER = getAllianceChooser();